spring-cloud-skipper-docs/src/main/asciidoc/three-hour-tour.adoc

```diff
@@ -204,44 +212,59 @@ The `apiVersion`, `kind` and `spec.resource` are required.
+Here are some examples:
 
 ----
 spec:
   resource: maven://org.springframework.cloud.samples:spring-cloud-skipper-samples-helloworld:1.0.0.RELEASE
+  resource: maven://{{maven-group-name}}:{{artifact-name}}:{{version}}
 ----
-The first part before the `:` is the Maven group name and the second part after the `:` is the artifact name.  The last part is the version.
+
+This shows the broad structure that maven uses. The first part before the `:` is the Maven group name and the second part after the `:` is the artifact name.  The last part is the version.
 
 ----
 spec:
   resource: docker:springcloud/spring-cloud-skipper-samples-helloworld:1.0.0.RELEASE
 ----
 This follows typical naming conventions of <user>/<repo>:<tag>.
 
-There is only one setting to specify with maven repositories to search.  This applies across all platform accounts.  By default the configuration
+There is only one setting to specify with maven repositories to search.  This applies across all platform accounts.  By default the configuration:
 
 ----
 maven:
   remoteRepositories:
     springRepo: https://repo.spring.io/libs-snapshot
 ----
 
-Is used.  You can specify other entries and also specify proxy properties.  This is currently best documented https://docs.spring.io/spring-cloud-dataflow/docs/1.3.0.M2/reference/htmlsingle/#getting-started-maven-configuration[here].
+is used.  You can specify other entries and also specify proxy properties.  This is currently best documented https://docs.spring.io/spring-cloud-dataflow/docs/1.3.0.M2/reference/htmlsingle/#getting-started-maven-configuration[here]. Essentially, this needs to be set a property in your launch properties or `manifest.yml` (when pushing to PCF) like so:
+
+----
+# manifest.yml
+...
+env:
+    MAVEN_REMOTE_REPOSITORIES_{{REPOSITORY_NAME}}_URL: https://repo.spring.io
+...
+----
 
 The metadata is used to help search for applications after they have been installed and is not available in Skipper M1.
 
 Currently, only `SpringCloudDeployerApplication` kind is supported which means the applications can be deployed into the target platforms only using their corresponding Spring Cloud Deployer implementations (CF, Kubernetes Deployer etc.).
+
 The `spec` contains the resource specification and the properties for the package.
+
 The `resource` represents the resource URI to download the application from. This would typically be a maven
 co-ordinate or a docker image URL.
+
 The `SpringCloudDeployerApplication` kind of application can have `applicationProperties` and `deploymentProperties` as the
 configuration properties.
+
 The application properties correspond to the properties for the application itself.
+
 The deployment properties correspond to the properties for the deployment operation performed by Spring Cloud Deployer implementations.
 
 [[package-values]]
-=== Package Values
+=== Package Values (Values.yml)
```

Comment:
Maybe we can use the actual file name with the lower case as values.yml as it might confuse users. Similarly for other headings as well. will fix when merging.

Reply:
:+1: